### ISSUE TITLE
Pi-hole server hostname occurs in top permitted domains table as domain

### ISSUE THREAD

**Issue by Haxy89** [pi-hole/pi-hole#2333]
![image](https://user-images.githubusercontent.com/37550896/43784822-4e3e125e-9a65-11e8-95c7-7abc8eaf9d11.png)

I have pi-hole on raspberrypi (hostname: raspberrypi) as you can see it occurs as domain in top permitted domains table

**Comment by DL6ER** [comment]
Click on the `raspberrypi` entry to see which one of your clients queried this domain. This may shine light on the reason why it is shown. I don't think this is a bug.

**Comment by Haxy89** [comment]
@DL6ER 
raspberrypi calls itself 
![image](https://user-images.githubusercontent.com/37550896/43785715-3abae16a-9a67-11e8-84cd-2c17dfef1fe6.png)

**Comment by DL6ER** [comment]
Okay, so this is no bug then. Something on your Pi-hole is querying `raspberrypi` and your Pi-hole replies. Just a wild guess: Did you maybe change the host name only recently and haven't restarted since then?

**Comment by Haxy89** [comment]
@DL6ER 
No, I did not change hostname, I restarted pi-hole before post this and did logs flush to see if bug happen again

**Comment by DL6ER** [comment]
I think some utils like `sudo` may be querying the local hostname so this is nothing you should worry about.

**Comment by Haxy89** [comment]
But this is something that occurs after update to pi-hole v4 so i think this is related to new version

**Comment by Exceen** [comment]
I have this behaviour as well, but that's because I use my Raspberry Pi as a SOCKS proxy and a VPN I guess.

**Comment by Mcat12** [comment]
Closing as there appears to be no bug. This can be reopened if a bug is found.

**Comment by Haxy89** [comment]
I don't understand, pi-hole server is not a domain, why it should be listed in this table...
Why pi-hole shows that localhost (pi-hole server) calls pi-hole server hostname, it does not make sense...

**Comment by dschaper** [comment]
What is the contents of the `/etc/hosts` file on the Pi-hole?

**Comment by Mcat12** [comment]
Both hostnames and domain names both go through the same channels in the DNS server when resolving the IP address, and there is no real difference between them to the DNS server. Because something queried `raspberrypi`, it got logged and is now shown on the dashboard.

**Comment by Haxy89** [comment]
@dschaper 
```
127.0.0.1       localhost
::1             localhost ip6-localhost ip6-loopback
ff02::1         ip6-allnodes
ff02::2         ip6-allrouters

```

**Comment by stitch10925** [comment]
I have the same problem, the pi-hole seems to be logging its own queries. This actually occurs when you have the pi-hole admin/dashboard open. There is a poll running in the background that updates the data on the UI. Ironically, this poll seems to be logged as query entry in the pi-hole:

![piadmin](https://user-images.githubusercontent.com/16646482/47047460-ececaa00-d197-11e8-8e00-09381416c092.png)

I actually came to report this, since this is really messing with the statistics. Pi-hole should not log its own requests, in my opinion.

**Comment by Mcat12** [comment]
Your device is not caching the DNS requests it makes, so they get sent to Pi-hole. Pi-hole then records the DNS queries it handles. Note: it does ignore `pi.hole` queries for this exact reason, but beyond that it can not just figure out what queries to ignore.

**Comment by stitch10925** [comment]
You are indeed correct. "pi.hole" does not get logged. Good to know. 

How about this: What if there was a text-box in the settings page in which you can fill in the FQDN of the pi-hole (or maybe the pi-hole can detect its own FQDN?). When this page is then browsed to, the user would be redirected to the pi.hole URL so that API calls are not logged?

That way you would not encounter this issue either: https://github.com/pi-hole/pi-hole/issues/2455

**Comment by Mcat12** [comment]
If the user just got redirected to http://pi.hole when visiting that hostname, then there would be no point in having that hostname because they could just go directly to http://pi.hole.

**Comment by stitch10925** [comment]
True, but if you use some kind of naming convention for your machines, it would be nice to not have to remember to use "pi.hole" as URL. Also, if you for some reason configured your router/client wrong and it is not pointing to the pi-hole, "pi.hole" will not resolve, but the FQDN probably will (even though the redirect will then also not work, unless you redirect it using IP address). Basically, redirecting the FQDN just making pi-hole more "idiot-proof" is suppose.
